recipe/meta.yaml

```diff
@@ -1,18 +1,18 @@
 {% set name = "widgetsnbextension" %}
-{% set version = "4.0.5" %}
+{% set version = "4.0.10" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/widgetsnbextension-{{ version }}.tar.gz
-  sha256: 003f716d930d385be3fd9de42dd9bf008e30053f73bddde235d14fbeaeff19af
+  sha256: 64196c5ff3b9a9183a8e699a4227fb0b7002f252c814098e66c4d1cd0644688f
 
 build:
   number: 0
   skip: True  # [py<37]
-  script: {{ PYTHON }} -m pip install . --no-deps -vvv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
```

Comment:
I know we pull the latest, but should we add something like:

`- jupyter-packaging >=0.10,<2` because of [this](https://github.com/jupyter-widgets/ipywidgets/blob/main/python/widgetsnbextension/pyproject.toml#L4)?

Reply:
indeed, I have updated the code, thank you!